packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -236,4 +240,68 @@ describe('logs', () => {
       expect(getCookie(SESSION_STORE_KEY)).toBeUndefined()
     })
   })
+
+  describe('session lifecycle', () => {
+    let clock: Clock
+    beforeEach(() => {
+      clock = mockClock()
+    })
+    afterEach(() => {
+      clock.cleanup()
+    })
+
+    it('sends logs without session id when the session expires ', () => {
+      setCookie(SESSION_STORE_KEY, 'id=foo&logs=1', ONE_MINUTE)
+      ;({ handleLog, stop: stopLogs } = startLogs(
+        initConfiguration,
+        { ...baseConfiguration, sendLogsAfterSessionExpiration: true },
+        () => COMMON_CONTEXT,
+        createTrackingConsentState(TrackingConsent.GRANTED)
+      ))
+      registerCleanupTask(stopLogs)
+
+      handleLog({ status: StatusType.info, message: 'message 1' }, logger)
+
+      // expire session
+      setCookie(SESSION_STORE_KEY, 'isExpired=1', ONE_MINUTE)
```

Comment:
**🥜 nitpick:** ‏this should be in a test helper, as we're doing it in many places